app/views/categories/index.html.erb

```diff
@@ -1,40 +1,44 @@
 <div class="menu">
-<%= render partial: "categories/navbar" %>
-<h2 class="text-light text-center">Categories</h2> 
-<% if(user_signed_in?) %>
-<%= button_to "Sign out", destroy_user_session_path, :method => :delete, data: {turbo: false}, class: "text-light text-center me-3"%>
-<% end %> 
+   <%= render partial: "categories/navbar" %>
+   <h2 class="text-light text-center">Categories</h2> 
+   <% if(user_signed_in?) %>
+     <%= button_to "Sign out", destroy_user_session_path, :method => :delete, data: {turbo: false}, class: "text-light text-center me-3"%>
+   <% end %> 
 </div>
 <div class="col-12 main_con">
-<% if @categories.length > 0 %>
-  
-<% if notice %>
-<p class="alert alert-success"><%= notice %></p>
-<% end %>
-<% if alert %>
-<p class="alert alert-danger"><%= alert %></p>
-<% end %> 
-<h3 class="row row-cols-2 p-3  bg-body opacity-75"><span>Total amount:</span> <span class="text-end">$<%= Category.total_amount(current_user.id)%></span></h3>
-<div>
-  <% @categories.each do |category| %> 
-    <div class="col-10 offset-1 row row-cols-3 shadow p-3 mt-3 bg-body opacity-75">
-      
-        <img src="<%= category.icon%>" >
-        <div class=" text-secondray" >
-         <%= link_to category.name , category_path(category.id) %><br/><br/>
-         <small ><%= category.created_at.strftime("%d %b %Y") %></small>
-        </div>
-        <div>
-        <p id="amount" class="text-end">$<%= category.entities.sum(:amount)%></p>
-        <%= button_to "Delete category", @category, method: :delete, class:"btn btn-primary btn-sm"%>
-        </div>
-        </div>
+  <div>
+   <% if @categories.length > 0 %> 
+      <% if notice %>
+      <p class="alert alert-success"><%= notice %></p>
       <% end %>
-</div> 
+      <% if alert %>
+      <p class="alert alert-danger"><%= alert %></p>
+      <% end %> 
+     <h3 class="col-12 row row-cols-2 p-3  bg-body opacity-75"><span>Total amount:</span> <span class="text-end">$<%= Category.total_amount(current_user.id)%></span></h3>
+    <div>
+        <% @categories.each do |category| %> 
+          <div class="col-10 bg-body offset-1 row row-cols-2 shadow p-3 mt-3  opacity-75">
+            
+              <img src="<%= category.icon%>" class="col-4">
+              <div class=" col-8">
+              <div class="  row row-cols-2 text-secondray " >
+              <%= link_to category.name , category_path(category.id), class: "color" %>
+              <p id="amount" class="text-end text-dark">$<%= category.entities.sum(:amount)%></p>
+              </div>
+              <hr />
+              <div class=" row row-cols-2">
+              <small ><%= category.created_at.strftime("%d %b %Y") %></small>
+               
+              <%= button_to "Delete_category", @category, method: :delete, class:"btn btn-primary btn-sm"%>
+              </div>
+              </div></div>
+            <% end %>
+    </div>
+  </div>
```

Comment:
- [ ] @AhmadiGu great work implementing the changes requested, however, the date doesn't fit in one line like in the design, also kindly consider removing the delete category button or making it fit the card `advice`(I would recommend you remove it since its not part of the requirements)
![image](https://user-images.githubusercontent.com/61408860/232227872-2af7d7be-a0a2-4e8c-a9c6-fa3c11c81c62.png)